geode-protobuf/src/main/java/org/apache/geode/internal/protocol/protobuf/v1/utilities/ProtobufEncodingTypes.java

```diff
@@ -28,22 +35,25 @@
   BOOLEAN(Boolean.class),
   DOUBLE(Double.class),
   FLOAT(Float.class),
-  BINARY(byte[].class);
+  BINARY(byte[].class),
+
+  // This will probably have to change once the protocol supports multiple object encodings.
+  PDX_OBJECT(PdxInstance.class);
```

Comment:
I kind of liked having this enum as a way to identify which types were en/de-codable directly by the protobuf code as opposed to those needing a custom encoder/decoder provided by the server (and also by the client).  Without that distinction, does it make sense to move this enum into the ProtobufSerializationService class?